vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -423,6 +423,7 @@ def execute_model(
         self._update_states(scheduler_output)
 
         model_input = self.prepare_model_input(scheduler_output)
+        self._mark_input_tensors(model_input)
```

Comment:
It looks like we'll be calling this for every model execution, but I think the dynamic/static settings are only for configuring the `torch.compile` for the model. It would be better if `_mark_input_tensors` only happened for the warmup.

Reply:
Ah, gotcha. I was also following the CB code block here and doing it always, I can try to pipe in some flag to see if we're in warmup mode to not do it on every forward pass and see what happens.

I'm not 100% sure though if this is supposed to be only at warmup time or if it is supposed to be on every forward pass